include/eigensolve_quda.h

```diff
@@ -21,12 +21,13 @@ namespace quda
 
     // Problem parameters
     //------------------
-    int nEv;          /** Size of initial factorisation */
-    int nKr;          /** Size of Krylov space after extension */
-    int nConv;        /** Number of converged eigenvalues requested */
-    double tol;       /** Tolerance on eigenvalues */
-    bool reverse;     /** True if using polynomial acceleration */
-    char spectrum[3]; /** Part of the spectrum to be computed */
+    int n_ev;            /** Size of initial factorisation */
+    int n_kr;            /** Size of Krylov space after extension */
+    int n_conv;          /** Number of converged eigenvalues requested */
+    int deflation_vecs; /** Number of converged eigenvalues to use in deflation */
```

Comment:
Given the convention of `n_ev`, `n_kr`, etc., perhaps a more consistent name for `deflation_vec` would be `n_ev_deflate`?